src/main/scala/nl/biopet/utils/biowdl/references/TestReference.scala

```diff
@@ -31,25 +31,29 @@ import nl.biopet.utils.biowdl.{fixtureFile, fixtureDir}
 trait TestReference extends Reference {
   def referenceSpecies = "test"
   def referenceName = "test"
-  def referenceFasta: File = fixtureFile("reference/reference.fasta")
+  def referenceFasta: File =
+    fixtureFile("references/test/reference/reference.fasta")
 
-  override def bwaMemFasta = Some(fixtureFile("reference/bwa/reference.fasta"))
+  override def bwaMemFasta =
+    Some(fixtureFile("references/test/reference/bwa/reference.fasta"))
 
   override def bowtieIndex: Option[File] =
-    Some(fixtureFile("reference/bowtie/reference"))
+    Some(fixtureFile("references/test/reference/bowtie/reference"))
 
   override def bowtie2Index: Option[File] =
-    Some(fixtureFile("reference/bowtie2/reference"))
+    Some(fixtureFile("references/test/reference/bowtie2/reference"))
 
   override def tophatIndex: Option[String] =
-    Some(fixtureDir + "reference/bowtie2/reference")
+    Some(fixtureDir + "references/test/reference/bowtie2/reference")
```

Comment:
I know this was already incorrect but I notice that `fixtureDir + "references/test/reference/bowtie2/reference"` is unsafe when the fixure dir is not ended with a slash. 
Can you convert this to this:
`fixtureDir + "/references/test/reference/bowtie2/reference"`

Reply:
ps: only the `+` is a problem. `Some(fixtureFile("reference/bowtie2/reference"))` is still fine for example